pcbdraw/populate.py

```diff
@@ -10,9 +10,17 @@
 from typing import List, Optional, Any, Tuple, Dict
 
 import click
-from mistune.plugins.table import plugin_table # type: ignore
-from mistune.plugins.footnotes import plugin_footnotes # type: ignore
 import mistune # type: ignore
+try:
+    from mistune.plugins.table import plugin_table # type: ignore
+    from mistune.plugins.footnotes import plugin_footnotes # type: ignore
+    InlineParser = mistune.inline_parser.InlineParser
+    HTMLRenderer = mistune.renderers.HTMLRenderer
+    mistune2 = True
```

Comment:
Instead of a variable in a global namespace, consider deciding on ` mistune.__version__`?

Reply:
The solution is highly inefficient and cryptic:
```python
tuple(map(int, mistune.__version__.split('.'))) != (0, 8, 4)
```
Not to mention that this is limited to a particular version, and not to the detection of the problem. 
Note that importing the `version` function from `packaging` (more inefficient, but not cryptic) will add undesired dependencies (I already dropped it for KiBot)

Reply:
Ok, I implemented individual detections in a767be5
In the class I check the v2 member exists and for the plugin hooks I just try them and ignore the error if they aren't defined

Reply:
I like the solution proposed in https://github.com/yaqwsx/PcbDraw/commit/a767be5a58e45f3bcaf5a6d36336fe8085e4f31c